tests/boulder-integration.sh

```diff
@@ -24,6 +24,7 @@ common() {
     letsencrypt_test \
         --authenticator standalone \
         --installer null \
+        --register-unsafely-without-email \
```

Comment:
This should be moved to `tests/integration/_common.sh` and have it replace the `--email ""` hack.